lib/libarbiter/player.h

```diff
@@ -45,6 +46,18 @@ class Player {
    */
   static Player FromJSON(std::string player_string);
 
+  /**
+   *  Serialises the Player to a line of CSV.
+   */
+  std::string ExportCSV() const;
+
+  /**
+   *  Serialises the Player to a JSON object.
+   *
+   *  The JSON is stored in a boost::property_tree::ptree.
+   */
+  boost::property_tree::ptree ExportJSON() const;
+
   boost::uuids::uuid id() const { return id_; }
   std::string name() const { return name_; }
   int elo() const { return elo_; }
```

Comment:
A system for rating the relative skill levels of chess players. https://en.wikipedia.org/wiki/Elo_rating_system